app/views/course/assessment/assessments/show.html.slim

```diff
@@ -71,6 +71,9 @@
     - @assessment.specific_conditions.each do |condition|
       = content_tag_for(:li, condition) do
         = condition.title
+    - if @assessment.start_at > Time.zone.now
```

Comment:
https://github.com/Coursemology/coursemology2/blob/d492ed917b46484f60b36d03351f371dbab6d1bf/spec/helpers/application_formatters_helper_spec.rb#L270
I think the spec is .start_at `>=` now instead of `>` @ekowidianto can you confirm?
Additionally, can I confirm that personalized timelines are handled?

Reply:
I did not explicitly account for personalized timelines, but looking at the code it seems like the strategies for personalized times would modify the `@assessment.start_at` time. I may be wrong on this, but let me check further.

Reply:
> is correct here since we are checking if it's satisfied.

@assessment.start_at is the standard time of an assessment and not modified. 

You can refer to the time_for method in the course/lesson_plan/item.rb model to get the correct timeline